test/upload_file/upload_file.wdl

```diff
@@ -0,0 +1,20 @@
+version 1.1
```

Comment:
@jdidion   Added minimal WDL file that uploads a file here. Still need to add integration test that calls dxCompiler with the flag.

Addressed other PR comments.

Reply:
Looks good. To add an integration test, go to scripts/run_tests.py and add the test name (without path or .wdl extension) to the appropriate test set.